### PR TITLE
[Baseline] Apply baseline to iceberg-orc #158

### PR DESCRIPTION
**Pull request by rdsr**
Apply baseline to iceberg-orc #158

### CODE REVIEW

File: orc/src/main/java/org/apache/iceberg/orc/ORC.java [hunk 1]
```diff
@@ -124,18 +123,18 @@ private ReadBuilder(InputFile file) {
...
-      this.length = length;
+    public ReadBuilder split(long newStart, long newLength) {
+      this.start = newStart;
+      this.length = newLength;
       return this;
     }
 
-    public ReadBuilder schema(org.apache.iceberg.Schema schema) {
-      this.schema = schema;
+    public ReadBuilder schema(org.apache.iceberg.Schema projectedSchema) {
```
Comment (rdblue) [inline orc/src/main/java/org/apache/iceberg/orc/ORC.java]: Nit: should probably be "projectSchema" not "projected". It doesn't make sense to use past tense; this is configuring a read that hasn't happened yet.

File: orc/src/main/java/org/apache/iceberg/orc/OrcFileAppender.java [hunk 2]
```diff
@@ -164,14 +164,15 @@ private static Writer newOrcWriter(OutputFile file,
...
 
     return writer;
   }
 
   @SuppressWarnings("unchecked")
-  private static <D> OrcValueWriter<D> newOrcValueWriter(TypeDescription schema,
-                                                         Function<TypeDescription, OrcValueWriter<?>> createWriterFunc) {
+  private static <D> OrcValueWriter<D> newOrcValueWriter(
+      TypeDescription schema,
+      Function<TypeDescription, OrcValueWriter<?>> createWriterFunc) {
```
Comment (rdblue) [inline orc/src/main/java/org/apache/iceberg/orc/OrcFileAppender.java]: I don't think this should have a newline. These should fit on one line, right?

**Comment by rdblue** [comment]
I'd merge this since there are only nits, but it conflicts with the baseline patch I just merged. Could you rebase and update?

File: orc/src/main/java/org/apache/iceberg/orc/OrcFileAppender.java (hunk 2 above)
Comment (rdsr) [reply orc/src/main/java/org/apache/iceberg/orc/OrcFileAppender.java]: Yes! I'll update

File: orc/src/main/java/org/apache/iceberg/orc/ORC.java (hunk 1 above)
Comment (rdsr) [reply orc/src/main/java/org/apache/iceberg/orc/ORC.java]: fixed

**Pull request merged**